templates/staff/application/detail.html

```diff
@@ -17,8 +17,8 @@
 {% block hero %}
 {% #staff_hero title="Application "|add:application.pk_hash %}
   <dl class="flex flex-col gap-2 mt-1">
-    {% if application.created_by %}
     <div>
+      {% if application.created_by %}
```

Comment:
This is an aside, but we always seem to have a value for `created_by` in the database. I know the field is nullable, but I wonder if that's something that needs to be changed?

Reply:
I don't think it is nullable. Unless I'm looking at the wrong `Application`?

https://github.com/opensafely-core/job-server/blob/3fabb37bc8e6413bd902394d073e217888a3cba3/applications/models.py#L38-L42

If it isn't, then the `if` is redundant, and I'll remove it in this PR. (I moved it in this PR, after all 🙂.)

Reply:
Ah yes I think you're right. I was looking at the database schema rather than the model definition and hadn't realised that they differ. Try removing it and see what happens :)

Reply:
This seems odd: there are many required fields (`null=False`, the default) whose columns aren't labelled as `not null`. I've asked a question on Slack.[^1]

[^1]: https://bennettoxford.slack.com/archives/C63UXGB8E/p1707744068865509